app/gui/view/welcome-screen/src/side_menu.rs

```diff
@@ -0,0 +1,73 @@
+//! Side menu for Welcome Screen.
+//!
+//! Side menu contains a list of available projects and "new project" button.
```

Comment:
Please check your docs with Gramarly – in several places there are small language issues. Here, for example, it should be `a "new project" button`.

Reply:
Fixed